.github/actions/find-pull-request/index.js

```diff
@@ -3,36 +3,53 @@ import * as core from '@actions/core'
 import * as github from '@actions/github'
 import { args } from './args.js'
 
+console.log(args.githubRef)
```

Comment:
Just confirming -- Were these console.logs meant to be here

Reply:
Oops. ty